### PR TITLE
Fontfamilybug

### PR DESCRIPTION
**Pull request by cybertheory**
## Description

Fixed bug with custom font family - Value was stored '"font-family"' instead of "font-family"

## Type of Change

- [ ] ❌ Breaking change
- [ ] ✨ New feature or improvement
- [x] 🛠️ Bug fix
- [ ] 🧹 Code refactor
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

## Ready Checklist

- [x] I've read the [Contribution Guide](https://github.com/schultek/jaspr/blob/main/CONTRIBUTING.md).
- [x] In case this PR changes one of the core packages, I've modified the respective **CHANGELOG.md** file using
      the [semantic_changelog](https://github.com/rrousselGit/semantic_changelog) format.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] I added myself to the AUTHORS file (optional, if you want to).

If you need help, consider asking for advice on the *#contribute* channel on [Discord](https://discord.gg/XGXrGEk4c6).

### CODE REVIEW

**Comment by schultek** [comment]
@cybertheory Please run `melos format` on this and your other prs. You probably have a different line length configured than the 120 that jaspr uses.

File: packages/jaspr/lib/src/ui/styles/properties/text.dart [hunk 1]
```diff
@@ -26,7 +26,7 @@ class FontFamily {
   final String value;
 
   /// Constructs a [FontFamily] from a custom font name
-  const FontFamily(String value) : value = '"$value"';
```
Comment (schultek) [inline packages/jaspr/lib/src/ui/styles/properties/text.dart:29]: Font family names are supposed to be wrapped in double quotes: https://developer.mozilla.org/en-US/docs/Web/CSS/font-family?retiredLocale=de

Comment (cybertheory) [reply packages/jaspr/lib/src/ui/styles/properties/text.dart:29]: The problem is it renders as a string within a string in inline styling, so I think it gets sanitized
Screenshot 2024-01-18 at 9 25 50 AM

Comment (cybertheory) [reply packages/jaspr/lib/src/ui/styles/properties/text.dart:29]: I know the font name is wrong but even with the correct name it seems to work only without the quotes

Comment (cybertheory) [reply packages/jaspr/lib/src/ui/styles/properties/text.dart:29]: https://stackoverflow.com/questions/32462939/inline-custom-css-fonts-in-html5

Comment (schultek) [reply packages/jaspr/lib/src/ui/styles/properties/text.dart:29]: Which browser are you using? For me it works also inline, even if its escaped.

Comment (cybertheory) [reply packages/jaspr/lib/src/ui/styles/properties/text.dart:29]: arc

Comment (schultek) [reply packages/jaspr/lib/src/ui/styles/properties/text.dart:29]: isn't that also chromium based?

Comment (cybertheory) [reply packages/jaspr/lib/src/ui/styles/properties/text.dart:29]: yup

**Comment by schultek** [comment]
Fixed this by using single quotes instead of double quotes around the font family name.

**Pull request closed without merging**